### PR TITLE
Update help text for dotnet-add-package

### LINKED ISSUE #5740: dotnet-add-package : Typo in description implies support for multiple packages
## Steps to reproduce
```
dotnet add package --help
.NET Add Package reference Command

Usage: dotnet add <PROJECT> package [arguments] [options]

Arguments:
  <PACKAGE_NAME>  Package references to add
  <PROJECT>       The project file to operate on. If a file is not specified, the command will search the current directory for one.
```

`dotnet add package Newtonsoft.Json Swagger`

## Expected  behavior
Adds Newtonsoft.Json and Swagger NuGet Package to project

## Actual behavior
Error:
“Please specify one package reference to add.”

## Environment data
```
Product Information:
 Version:            1.0.0-rc4-004777
 Commit SHA-1 hash:  18c01886cd

Runtime Environment:
 OS Name:     Mac OS X
 OS Version:  10.12
 OS Platform: Darwin
 RID:         osx.10.12-x64
 Base Path:   /usr/local/share/dotnet/sdk/1.0.0-rc4-004777
```

### PR DESCRIPTION
**Pull request by spboyer**
skipciplease

- typo "targetting"
- Fixes #5740

### CODE REVIEW

File: src/dotnet/MsbuildProject.cs [hunk 1]
```diff
@@ -174,7 +174,7 @@ public bool CanWorkOnFramework(NuGetFramework framework)
             return false;
         }
 
-        public bool IsTargettingFramework(NuGetFramework framework)
+        public bool IstargetingFramework(NuGetFramework framework)
```
Comment (nil4) [inline src/dotnet/MsbuildProject.cs]: `IsTargeting` (uppercase T)?

Comment (spboyer) [inline src/dotnet/MsbuildProject.cs]: This should be CamelCase. I'll fix.

File: src/dotnet/commands/dotnet-add/dotnet-add-package/xlf/LocalizableStrings.pl.xlf [hunk 2]
```diff
@@ -14,7 +14,7 @@
         <note />
       </trans-unit>
       <trans-unit id="AppHelpText">
-        <source>Package references to add</source>
+        <source>Package reference to add</source>
         <target state="translated">Odwołania do pakietów, które mają zostać dodane</target>
```
Comment (krwq) [inline src/dotnet/commands/dotnet-add/dotnet-add-package/xlf/LocalizableStrings.pl.xlf]: (not related to this PR) this translation couldn't be any worse cc: @blackdwarf - I just looked at the Polish translation and it is quite not right - while literally it is translated fairly ok, semantically and contextually it doesn't make sense

**Comment by mairaw** [comment]
Curiosity: are we supposed to touch the XLIFF files when making fixes to localizable strings? I thought those were generated as part of the handoff/handback process.

File: src/dotnet/commands/dotnet-add/dotnet-add-package/LocalizableStrings.cs [hunk 3]
```diff
@@ -9,11 +9,11 @@ internal class LocalizableStrings
 
         public const string AppDescription = "Command to add package reference";
 
-        public const string CmdPackageDescription = "Package references to add";
+        public const string CmdPackageDescription = "Package reference to add";
```
Comment (mairaw) [inline src/dotnet/commands/dotnet-add/dotnet-add-package/LocalizableStrings.cs]: we need to start adding the missing punctuation too. Also wouldn't this read better as "The package reference to add."?

File: src/dotnet/commands/dotnet-add/dotnet-add-package/xlf/LocalizableStrings.pl.xlf (hunk 2 above)
Comment (blackdwarf) [inline src/dotnet/commands/dotnet-add/dotnet-add-package/xlf/LocalizableStrings.pl.xlf]: @krwq ok, let me see if we can somehow change that post v-1.

**Comment by blackdwarf** [comment]
@mairaw quite right, these are auto-generated.

**Comment by piotrpMSFT** [comment]
@spboyer can your revert the xlf files please? These get automatically generated. @mairaw how do we best communicate the associated xlf changes to get the database updated?

**Comment by mairaw** [comment]
@piotrpMSFT I just have a basic idea of the localization workflow. Wouldn't that question be more appropriate to the Loc person working on the project?

**Comment by spboyer** [comment]
@piotrpMSFT ok, all done. Should be able to merge and regen the .xlf files now.

**Pull request merged**